renku/models/provenance/activities.py

```diff
@@ -418,12 +417,12 @@ def _load(step):
                 return step.run
 
             if self.commit:
+                import yaml
```

Comment:
Is this imported second time or I'm blind?

Reply:
I have removed the global one.